cnf-certification-test/certification/suite.go

```diff
@@ -43,51 +43,55 @@ const (
 var _ = ginkgo.Describe(common.AffiliatedCertTestKey, func() {
 	logrus.Debugf("Entering %s suite", common.AffiliatedCertTestKey)
 	var env provider.TestEnvironment
+	var certificator certdb.ResourceCertificator
 	ginkgo.BeforeEach(func() {
+		var err error
 		env = provider.GetTestEnvironment()
-		err := api.LoadCatalog(env.GetOfflineDBPath())
+		certificator, err = certdb.GetCertificator(env.GetOfflineDBPath())
 		if err != nil {
-			logrus.Infof("Offline DB not available, err: %v", err)
+			errMsg := fmt.Sprintf("Cannot access the certification DB, err: %v", err)
+			logrus.Errorf(errMsg)
+			ginkgo.Fail(errMsg)
```

Comment:
Maybe we already have this pattern somewhere else, but I don't think it's necessary to log the same msg with logrus and  with the ginkgo.Fail(). Both will appear in the stdout, so, in this cases, I think that only the ginkgo.Fail(errMsg) is needed.

Reply:
Done.